.travis.yml

```diff
@@ -14,7 +14,7 @@ bundler_args: --without development --deployment
 
 before_script:
   - cd dashboard
-  - bundle exec rake --trace db:create db:schema:load
+  - bundle exec rake --trace db:create db:schema:load seed:test
```

Comment:
I think we need db:create because there are no databases in travis -- I can see if `db:test:prepare` works